src/components/SocialBtns/SocialBtns.js

```diff
@@ -1,24 +1,32 @@
+/* global FB, location */
 import React, { PropTypes } from 'react';
 import FaFacebookSquare from 'react-icons/lib/fa/facebook-square';
 import FaTwitter from 'react-icons/lib/fa/twitter';
 import styles from './SocialBtns.sass';
 
 const SocialBtns = ({
-  fbLink = 'https://www.facebook.com',
-  twitterLink = 'https://twitter.com',
+  fbLink = `https://www.facebook.com/sharer/sharer.php?u=${location.href}`,
```

Comment:
It sounds like the only way to get the Facebook parser to recognize dynamically set og meta tags might be to render on the server ([based on this](https://github.com/nfl/react-helmet/issues/213)). I admittedly don't know a whole lot about server-rendered React. Is that the current set up?